iOS Example Frame SPM/iOS Example Frame Integration Tests/CardTypeTokenCreationTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  iOS_Example_Frame_Integration_Tests.swift
+//  iOS Example Frame Integration Tests
+//
+//  Created by Okhan Okbay on 17/07/2023.
+//  Copyright © 2023 Checkout. All rights reserved.
+//
+
+import XCTest
+import Frames
+
+final class iOS_Example_Frame_Integration_Tests: XCTestCase {
+
+    func testTokenCreation() throws {
+        // VISA
+        verifyCardTokenCreation(cardNumber: "4485040371536584", securityCode: "123")
+
+        // MasterCard
+        verifyCardTokenCreation(cardNumber: "5588686116426417", securityCode: "123")
```

Comment:
Would it be nice to add all the card constants in one place and you can reuse it elsewhere if needed in future or for other tests

Reply:
This is updated now. How's the new look?